dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: bidder
+title: Nexverse
+description: Prebid Nexverse Bidder Adapter
+biddercode: nexverse
+aliasCode: nexverse
+tcfeu_supported: true
+dsa_supported: true
+gvl_id: none
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: false
+userId: all
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: true
+pbjs: true
+pbs: true
```

Comment:
Do you have a Prebid Server adapter? I don't see a PR. Please change this to false.